documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc

```diff
@@ -201,6 +201,30 @@ Both the local and the backup call(if the "backupFailurePolicy" is set to "FAIL"
 ===== Asynchronous local cluster
 In the case of asynchronous local clusters, the backup data is sent during the commit phase. If the backup call fails and the "backupFailurePolicy" is set to "FAIL" then the user is notified through an exception.
 
+==== Expiring cache entries across sites
+You can configure {brandname} to expire entries, which controls the amount of
+time entries remain in the cache.
+
+* `lifespan` expiration is suitable for cross-site replication because entries
+can expire on either site without the need to synchronize system time between
+sites.
+
+* `max-idle` expiration is not suitable for cross-site replication because
+{brandname} needs to send requests across sites to determine if cache entries
+have reached the idle timeout. This also requires sites to have synchronized
+system times regardless of time zone.
+
+[IMPORTANT]
+====
+When expiring entries, {brandname} replicates commands across sites for both
+the `lifespan` and `max-idle` parameters. However, these commands have no
+effect. When a large volume of entries expire, {brandname} replicates lots of
+unnecessary expiration commands across sites.
+====
+
+* `interval` expiration is suitable for cross-site replication because
```

Comment:
What is this sorry?

Reply:
@wburns I included the `interval` parameter alongside `lifecycle` and `max-idle` for completeness. it's in the schema docs: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/Configuration/infinispan-config-9.4.html 


interval | long | 60000 | Interval (in milliseconds) between  subsequent runs to purge expired entries from memory and any cache  stores. If you wish to disable the periodic eviction process altogether,  set interval to -1.

Reply:
Okay, it is a bit confusing to me, to be honest. This variable has nothing to do with xsite and just controls how often the reaper runs on a given node (it isn't even specific to a cluster).

Reply:
@wburns I'll just remove this because `interval` isn't replicated